README.md

```diff
@@ -30,6 +30,16 @@ And then execute:
 $ bundle install
 ```
 
+Add this line to your application's Rakefile:
+
+```ruby
+begin
+  require "rails-mermaid_erd"
+rescue LoadError
+  # Do nothing.
+end
```

Comment:
Another possible implementation would be as follows

```ruby
if Rails.env.development?
  require “rails-mermaid_erd”
end
````

I did not adopt this method because of the redundancy of having the information “use only in the development gem group” expressed in two places, in the Gemfile and in the Rakefile.